pom.xml

```diff
@@ -207,76 +221,6 @@
                 </plugins>
             </build>
         </profile>
-        <profile>
-            <id>directory</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-assembly-plugin</artifactId>
-                        <version>3.6.0</version>
-                        <configuration>
-                            <appendAssemblyId>false</appendAssemblyId>
-                            <descriptors>
-                                <descriptor>src/main/assembly/assembly.xml</descriptor>
-                            </descriptors>
-                        </configuration>
-                        <executions>
-                            <execution>
-                                <goals>
-                                    <goal>single</goal>
-                                </goals>
-                                <phase>install</phase>
-                            </execution>
-                        </executions>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-source-plugin</artifactId>
-                        <version>3.3.0</version>
-                        <executions>
-                            <execution>
-                                <id>attach-sources</id>
-                                <phase>verify</phase>
-                                <goals>
-                                    <goal>jar-no-fork</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
Javadoc and jar plugins might be needed, why you want to delete them?

Reply:
I think they are both needed. But not in the directory profile - they have to run always.

Reply:
Also directory profile seems not needed at all - we are not going to deploy this plugin into Vaadin Directory.

Reply:
Thanks for the review! I may miss something :), but I see that I removed the `directory` profile fully from the `pom.xml. `

And I am on the same side about the removed plugins from the `directory` profile, as my comment shows above: 

I am thinking also about adding `javadoc` and `jar` plugins profiles back, and those could be added under the <build> without any <profile> for always running.

Let's catchup on this tomorrow daily. Take care until then!

Reply:
Ah, indeed, you removed it. Sorry, it was the end of the day :)